api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/ObjectPropertyTemplateModel.java

```diff
@@ -386,6 +386,20 @@ protected String getObjectKey() {
         return objectKey;
     }
 
+    protected boolean isAuthorizedToDisplay(VitroRequest vreq, String objectUri) {
+        boolean isAuthorizedToDisplay = false; 
+        AccessObject ao;
+        if (isFaux()) {
```

Comment:
This may be more elegant with a ternary.

```
AccessObject ao = isFaux()
  ? new FauxObjectPropertyStatementAccessObject(vreq.getJenaOntModel(), subjectUri, fauxProperty, objectUri)
  : new ObjectPropertyStatementAccessObject(vreq.getJenaOntModel(), subjectUri, property, objectUri);
```

Reply:
Fixed